src/main/java/org/breedinginsight/api/v1/controller/ProgramController.java

```diff
@@ -399,4 +405,20 @@ public HttpResponse archiveProgramLocation(@PathVariable UUID programId,
         }
     }
 
+    @Get("/programs/{programId}/observation-levels")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ProgramSecured(roleGroups = {ProgramSecuredRoleGroup.ALL})
+    public HttpResponse<Response<DataResponse<ProgramObservationLevel>>> getProgramObservationLevels(@PathVariable UUID programId)
+            throws DoesNotExistException {
+        List<ProgramObservationLevel> programObservationLevels = programObservationLevelService.getByProgramId(programId);
+
+        List<org.breedinginsight.api.model.v1.response.metadata.Status> metadataStatus = new ArrayList<>();
```

Comment:
You can remove the package info, and just leave it as `Status`

Reply:
changed and pushed